### ISSUE TITLE
Emacs installing problem in cygwin 

### ISSUE THREAD

**Issue by gadimli93** [snu-sf-class/pl201602#6]
Hello~
I successfully followed all instruction and until Emacs installation.

I'm using my cygwin in windows, 
i successfully cloned:
https://github.com/ProofGeneral/PG ~/.emacs.d/lisp/PG  

but when i tried to open the directory it says that " no such file or directory"

![capture](https://cloud.githubusercontent.com/assets/11403950/18380556/db0e6a40-76b2-11e6-904c-5885756e9426.JPG)

Looking forward you reply.
Thanks

**Comment by jeehoonkang** [comment]
What is the result of `echo $HOME`?  I suspect `/home/...` is not your home directory.

**Comment by gadimli93** [comment]
i checked home directory, its mine..

**Comment by jeehoonkang** [comment]
- please do not upload an image of code blocks.  It is hard to parse.  Use block quotations.
- what is the result of `echo $HOME`?

**Comment by jeehoonkang** [comment]
- What is the results of `ls ~`, `ls ~/.emacs.d`, `ls ~/.emacs.d/lisp`, and `ls ~/.emacs.d/lisp/PG`?

**Comment by jeehoonkang** [comment]
- Also, it seems from the `fatal ...` that the clone is not successful..

**Comment by gadimli93** [comment]
for the result of the echo $HOME, is 

> Rubis-N.Gadimli@DESKTOP-NVEELQL ~
> $ echo $HOME
> /home/Rubis-N.Gadimli

---

result for ls ~

> Rubis-N.Gadimli@DESKTOP-NVEELQL ~
> $ ls ~
> index.html  index.html.1 

---

and for the rest commands about ls ~/.emacs.d and etc:

> Rubis-N.Gadimli@DESKTOP-NVEELQL ~
> $ ls ~/.emacs.d
> ls: cannot access '/home/Rubis-N.Gadimli/.emacs.d': No such file or directory
> 
> Rubis-N.Gadimli@DESKTOP-NVEELQL ~
> $ ls ~/.emacs.d/lisp
> ls: cannot access '/home/Rubis-N.Gadimli/.emacs.d/lisp': No such file or directory

---

no, i dont think so, because when cloned the repository, there was written that 100% done,succesfully
i think i'm going delete everything and download/install all of them again

**Comment by jeehoonkang** [comment]
let's do it:

```
mkdir -p ~/.emacs.d/lisp
git clone https://github.com/ProofGeneral/PG ~/.emacs.d/lisp/PG
```

**Comment by gadimli93** [comment]
unfortunately this is also not worked

> Rubis-N.Gadimli@DESKTOP-NVEELQL ~
> $ mkdir -p ~/.emacs.d/lisp
> 
> Rubis-N.Gadimli@DESKTOP-NVEELQL ~
> $ git clone https://github.com/ProofGeneral/PG ~/.emacs.d/lisp/PG
> fatal: destination path '/home/Rubis-N.Gadimli/.emacs.d/lisp/PG' already exists and is not an empty directory.

**Comment by jeehoonkang** [comment]
How about:

```
git clone https://github.com/ProofGeneral/PG
mv PG ~/.emacs.d/lisp
```

?

**Comment by gadimli93** [comment]
it worked :) now i have access to the directory
thank you
